environments.ini

```diff
@@ -1675,3 +1676,62 @@ build_flags =
 ;  '-DsimplePublishing=true'
 custom_description = BLE Gateway using ethernet, requires PIO configuration
 custom_hardware = ThingPulse gateway single ESP32
+
+[env:d1mini-temt6000-ls]
+platform = ${com.esp8266_platform}
+board = d1_mini
+lib_deps =
+  ${com-esp.lib_deps}
+  ${libraries.wifimanager8266}
+  ${libraries.esp8266_mdns}
+build_flags =
+  ${com-esp.build_flags}
+  '-UZmqttDiscovery'          ; disables MQTT Discovery
+;  '-UZmqttDiscovery="HADiscovery"'
+  '-DvalueAsATopic=true'       ; MQTT topic includes model and device
+  '-DGateway_Name="OpenMQTTGateway_TEMT6000_LS"'
+  '-DZsensorTEMT6000="TEMT6000"'
+board_build.flash_mode = dout
+custom_description = Light sensor gateway 
+custom_hardware = Light sensor first version
+
+[env:esp01-hcsr501-ms]
+platform = ${com.esp8266_platform}
+board = esp01_1m
+board_build.ldscript = eagle.flash.1m64.ld
+lib_deps =
+  ${com-esp.lib_deps}
+  ${libraries.wifimanager8266}
+  ${libraries.unifiedsensor}
+  ${libraries.esp8266_mdns}
+build_flags =
+  ${com-esp.build_flags}
+  '-UZmqttDiscovery'          ; disables MQTT Discovery
+;  '-UZmqttDiscovery="HADiscovery"'
+  '-DvalueAsATopic=true'       ; MQTT topic includes model and device
+  '-DHCSR501_LED_NOTIFY_GPIO=2'
+  '-DHCSR501_LED_ON=LOW'
+  '-UMQTTsetMQTT' ;We remove this function to have sufficient FLASH available for OTA, you should also use ESPWifiManualSetup and 'board_build.ldscript = eagle.flash.1m64.ld' to save flash memory and have OTA working
+  '-UMQTT_HTTPS_FW_UPDATE' ;We remove this function to have sufficient FLASH available for OTA, you should also use ESPWifiManualSetup and 'board_build.ldscript = eagle.flash.1m64.ld' to save flash memory and have OTA working
+;  '-DGateway_Name="OpenMQTTGateway_CSR501_MS"'
+  '-DZsensorHCSR501="HCSR501"'
+  '-DHCSR501_GPIO=0'
+  '-DGateway_Name="OpenMQTTGatewayMS"'
+  '-DESPWifiManualSetup=true'
+  '-Dwifi_ssid=""'
+  '-Dwifi_password=""'
+  '-DMQTT_USER=""'
+  '-DMQTT_PASS=""'
+  '-DMQTT_SERVER="..."'
+board_build.flash_mode = dout
+custom_description = Motion sensor gateway 
+custom_hardware = Motion sensor first version
+
+[env:esp01-hcsr501-ms-ota-test]
+extends = env:esp01-hcsr501-ms
+upload_protocol = espota
+upload_port = 127.0.0.1
+upload_flags =
+  --auth=OTAPASSWORD
+  --port=8266
+  
```

Comment:
Could you add ` '-DZsensorTEMT6000="TEMT6000"'` to `esp32dev-all-test` so that the continuous integration will check that it builds with ESP32?

We should not add three other environments. The users should store this kind of custom environment in a prod_env.ini
By adding the module into the `nodemcu-all-test` and the `esp32dev-all-test` we verify that the module builds correctly and can be used by changing the configuration of the gateway through Platformio.
I would like to avoid being in a situation on which we store all possible configuration in the web upload and make it difficult for other users to find the right binary.

Reply:
All good, done.